### DISCUSSION TITLE
We need to change the build region

### DISCUSSION THREAD

**Discussion by yiseungyun** [vercel/community#6253] [Help]
### Summary

I deployed the next.js project using vercel, and I am currently a resident of Korea. The regional setting of the serverless function can be changed and changed to Korea, but **there was a problem that it was impossible to change the address where static files such as images were provided.** The image files continue to be received from the United States, which is causing it to face a problem that is significantly slow to load image files. I know that it automatically connects to a nearby area, but I wonder why it is connected to the United States. When I deploy my project, the address to be built is also the United States, so is it impossible to change it to an address that is close to Korea or Korea? It would be nice if a setting to change this location could be added.

### Steps to Reproduce

<img width="1254" alt="스크린샷 2024-03-25 오후 1 17 54" src="https://github.com/vercel/community/assets/73896327/5a3be9d5-7d78-4fa1-82e3-c2b78515380d">

<img width="671" alt="스크린샷 2024-03-25 오후 2 20 03" src="https://github.com/vercel/community/assets/73896327/71dedb39-b581-4454-afcc-c4c5a96a4e3c">

> Since static files are imported from the United States, they are much slower after deployment than when tested in local, which is inconvenient.

**Comment by versecafe** [comment]
@yiseungyun Is this an issue for you if you go through the pages a few times and then wait a minute for them to be cached globally, Vercel should be caching your static assets across their global CDN only first serve should come from DC, also have you changed your function region over?

### Set function region to ICN1 for Korea
<img width="1274" alt="image" src="https://github.com/vercel/community/assets/147033096/50452b86-0e70-44c0-8590-2c4a426b095f">
According to docs the files should be cached after first serve for 31 days:
<img width="1274" alt="image" src="https://github.com/vercel/community/assets/147033096/cb9c24aa-deb2-4c6c-b2b8-183b72cce6a2">

**Reply by yiseungyun** [reply]
I made a mistake! I read your answer and also confirmed that the function region is already set to Korea. And I checked again that the image loading was slow even though the image was preloaded using the image tag provided by next/image. But when I looked at it again, I realized that the image was preloaded and imported at a high speed, and it was slow when I imported the image from the subsequent request. I need to fix this part, but I think I was confused because this image was continuously imported from the United States. Thank you so much for your answer! I think I can fix the problem well! 😹
